### PR TITLE
📖 Updates to the release issue template

### PR DESCRIPTION
**Pull request by danielrozenberg**
* Switch explanation to using pre-existing Nightly releases instead of cutting a `canary` branch
* Consistent capitalization of release channels

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/release-tracking-issue.md [hunk 1]
```diff
@@ -24,30 +25,28 @@ with the release so please keep this issue up to date:
...
+- If this release is promoted to Stable on the second, third, fourth, or fifth Monday of a given month, it is not an LTS release candidate (but will be included in a later LTS release).
 
 Based on the above, if this release must be promoted to LTS, copy-paste the following checkbox into the list above.
 
 - [ ] Release <RELEASE_NUMBER> promoted to LTS (<CL submit time>)
 
 If you perform cherry picks, add/update the checkboxes above as needed e.g.
 
-- [ ] Release `[<CHERRY_PICK_RELEASE_NUMBER>](...)` created with cherry picks
+- [ ] Release [<CHERRY_PICK_RELEASE_NUMBER>](...) created with cherry picks
```
Comment (rsimha) [inline .github/ISSUE_TEMPLATE/release-tracking-issue.md:49]: I'd go a step further and pre-fill this with the incomplete URL.

```markdown
[ ] Release [<CHERRY_PICK_RELEASE_NUMBER>](https://github.com/ampproject/amphtml/releases/tag/<CHERRY_PICK_RELEASE_NUMBER>) created with cherry picks
```

File: .github/ISSUE_TEMPLATE/release-tracking-issue.md [hunk 2]
```diff
@@ -24,30 +25,28 @@ with the release so please keep this issue up to date:
...
 - [ ] Release <CHERRY_PICK_RELEASE_NUMBER> promoted to Experimental and Beta channels
```
Comment (rsimha) [inline .github/ISSUE_TEMPLATE/release-tracking-issue.md:50]: Add `(<CL submit time>)` at the end of this line

```markdown
[ ] Release <CHERRY_PICK_RELEASE_NUMBER> promoted to Experimental and Beta channels (<CL submit time>)
```

Comment (danielrozenberg) [reply .github/ISSUE_TEMPLATE/release-tracking-issue.md]: Done

File: .github/ISSUE_TEMPLATE/release-tracking-issue.md (hunk 1 above)
Comment (danielrozenberg) [reply .github/ISSUE_TEMPLATE/release-tracking-issue.md]: Done + added a placeholder to add the cherry pick issue # number

**Pull request merged**